modules/social_features/social_group/social_group.install

```diff
@@ -339,7 +338,6 @@ function social_group_update_8010() {
     else {
       user_role_grant_permissions($role->id(), [
         'create public_group group',
-        'view group stream page',
```

Comment:
We shouldn't be changing previous update hooks. Adding a non-existant permission in Drupal doesn't give any errors. If we want to explicitly remove the permission from a role then we should write an update hook anyway.

Reply:
You are right - adding non-existant permission in Drupal doesn't give any errors, but adding these permissions was not correct. I think it makes sense to remove it, to avoid confusion for a developer who will read this. Anyway, with removing this line or not - it doesn't change anything functionally.